mne/viz/ica.py

```diff
@@ -267,9 +269,13 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
 
     for exc_label, ii in zip(exclude_labels, picks):
         if exc_label is not None:
-            if labels is not None:
-                exc_label = exc_label.split(' - ')[1]
-            color = label_colors[exc_label]
+            # create look up for color ...
+            if ' - ' in exc_label:
```

Comment:
So we don't need the `labels is not None` check anymore, right? It looks like we don't based on how it's constructed above, just want to double-check.